src/config.h

```diff
@@ -0,0 +1,50 @@
+#ifndef CONFIG_H_PIBRI8ND
+#define CONFIG_H_PIBRI8ND
```

Comment:
I am really curious about where this came from. Even though I have no issue, for consistency sake, we should probably use `DRAFTER_CONFIG_H`.